pkg/api/types.go

```diff
@@ -50,12 +50,12 @@ type UpdateTaskRequest struct {
 type TaskKind string
 
 const (
-	TaskKindDeno   TaskKind = "deno"
-	TaskKindDocker TaskKind = "docker"
-	TaskKindGo     TaskKind = "go"
-	TaskKindManual TaskKind = ""
-	TaskKindNode   TaskKind = "node"
-	TaskKindPython TaskKind = "python"
+	TaskKindDeno        TaskKind = "deno"
+	TaskKindDockerfile  TaskKind = "dockerfile"
```

Comment:
```suggestion
	TaskKindDockerFile  TaskKind = "docker.file"
```

just for readability, feel free to change the `.` to a `_` if you want to.

Reply:
yeah it's sort of silly because "Dockerfile" is a thing, but i'll do docker_file + docker_image

Reply:
meh actually I'm not sure about this change, will discuss https://airplanedev.slack.com/archives/C01DYGWC3P0/p1622148169011900